otter/test/convergence/test_selfheal.py

```diff
@@ -30,72 +30,54 @@ class SelfHealTests(SynchronousTestCase):
     def setUp(self):
         self.clock = Clock()
         self.log = mock_log()
-        self.ggtc = patch(
-            self, "otter.convergence.selfheal.get_groups_to_converge",
-            side_effect=intent_func("ggtc"))
-        self.lb, lock = create_fake_lock()
-        self.s = sh.SelfHeal("disp", 300, self.log, self.clock, "cf",
-                             lock=lock)
-
-    def test_setup_again(self):
-        """
-        Calls _setup_if_locked at every interval
-        """
-        self.s._setup_if_locked = mock.Mock()
-        self.s.startService()
-        self.s._setup_if_locked.assert_called_once_with("cf", 300)
-        self.clock.advance(300)
-        self.assertEqual(self.s._setup_if_locked .call_count, 2)
+        self.patch(sh, "get_groups_to_converge", intent_func("ggtc"))
+        self.patch(sh, "check_and_trigger", lambda t, g: t + g)
+        self.s = sh.SelfHeal(self.clock, base_dispatcher, "cf", 300.0,
+                             self.log)
+        self.groups = [
+            {"tenantId": "t{}".format(i), "groupId": "g{}".format(i)}
+            for i in range(5)]
+
+    def test_call(self):
+        """
+        ``self.s()`` will setup convergences to be triggered over specified
```

Comment:
This should probably be documented as ``:obj:`SelfHeal` `` rather than ``self.s()``, since it's a reference to the behavior of the `SelfHeal` class.